frontend/src/metabase-types/api/settings.ts

```diff
@@ -129,12 +129,52 @@ export type LoadingMessage =
 
 export type TokenStatusStatus = "unpaid" | "past-due" | "invalid" | string;
 
+const tokenStatusFeatures = [
```

Comment:
Where did you get this list from?

Reply:
These are coming from `/api/premium-features/token/status`